Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -72,7 +72,7 @@ type BocAutoCompleteReferenceValue_Options = {
     max: number;
     isAutoPostBackEnabled: boolean;
     extraParams: Dictionary<unknown>;
```

Comment:
We should rename the file as a separate commit at the end. maybe BocAutoCompleteReferenceValue.UI.ts or BocAutoCompleteReferenceValue.Combobox.ts or something along those lines. Ideas welcome

Reply:
I'll do this once everything else is settled

Reply:
I would go with `BocAutoCompleteReferenceValue.UI.ts`